lib/plugins/filter/before_post_render/indented_code_block.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { escapeHTML, stripIndent } = require('hexo-util');
+
+const rIndentedCodeBlock = /(^|(?:^|\n)(?:(?: {0,3}>){1,3}[^\S\r\n])?[^\S\r\n]*\n)((?:((?:(?: {0,3}>){1,3}[^\S\r\n])?)( {4}|\t)([^\n]*?)(\n|$))+)/g;
```

Comment:
```suggestion
const rIndentedCodeBlock = /(^|(?:^|\n)(?:(?: {0,3}>){1,3}[^\S\r\n])?[^\S\r\n]*\n)((?:((?:(?: {0,3}>){1,3}[^\S\r\n])?)( {4}|\t)([^\n]*?)(\n|$))+)/gm;
```

Should `gm` to be used?

```
test test

    hello world

    hello world

test test
```

There is a case that second `hello world` won't be matched.

Reply:
With the option `m`, indented line becomes target even if its preceding line is not a blank line.

I adjusted regexp, and add test case.